tests/test_progress.py

```diff
@@ -79,6 +79,28 @@ class FakeTask(Task):
     assert str(text) == "0:01:00"
 
 
+def test_download_progress_uses_decimal_units() -> None:
```

Comment:
Better, but too complex for a test.

I see you're trying to be thorough (which is good!) but a test should be no more complex that is needed to catch a bug.

Have a look at `test_time_remaining_column`, you'll notice that there are no loops, just assignments and method calls, but that's enough to execute every line of the changed code. I think you can do something similar here...

Reply:
Ok, makes sense, I thought I will add test all the symbols. Should I just create a task say half completed and just assert the rendered string?

Reply:
Sounds reasonable 👍